dask/array/core.py

```diff
@@ -536,7 +535,9 @@ def blockdims_from_blockshape(shape, blockshape):
     ((4, 4, 2), (3, 3, 3, 1))
     """
     if blockshape is None:
```

Comment:
looks like this function and its arguments needs a rename. perhaps `expand_chunks`?